Pipfile

```diff
@@ -0,0 +1,26 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+jikanpy = "*"
+pytest = "*"
+pytest-datadir = "*"
+pytest-factoryboy = "*"
+factory-boy = "*"
+vcrpy = "*"
+mypy = "*"
+mypy-extensions = "*"
+sqlalchemy-stubs = "*"
+black = "*"
+PyMySQL = "*"
+SQLAlchemy = "*"
```

Comment:
Shouldn't jikanpy, PyMySQL, and SQLAlchemy be in `[packages]`? Someone just running the scripts need those dependencies, but only developers need all the testing and type checking dependencies.

Reply:
so I think it actually needs to be added to both...i think that's how pipenv works. i just didn't get around to updating `[packages]` yet.

Reply:
indeed. was doing some testing of ci. this should be fixed now